src/MySqlConnector/Protocol/ProtocolCapabilities.cs

```diff
@@ -147,4 +147,14 @@ internal enum ProtocolCapabilities : ulong
 	/// Support of array binding.
 	/// </summary>
 	MariaDbStatementBulkOperations = 0x4_0000_0000,
+
+	/// <summary>
+	/// metadata extended information
+	/// </summary>
+	MariaDbExtendedTypeInfo = 0x8_0000_0000,
+
+	/// <summary>
+	/// permit metadata caching
+	/// </summary>
+	MariaDbCacheMetadata = 0x16_0000_0000,
```

Comment:
```suggestion
	MariaDbCacheMetadata = 0x10_0000_0000,
```

I assume this should just be the single bit?

Reply:
corrected with latest push